src/bundles/analytics.js

```diff
@@ -49,15 +49,20 @@ import { ACTIONS as EXP } from './experiments'
  * @property {'ANALYTICS_ADD_CONSENT'} type
  * @property {{name:string}} payload
  *
+ * @typedef {Object} ToggleAskToEnable
+ * @property {'ANALYTICS_ASK_TO_ENABLE'} type
```

Comment:
```suggestion
 * @property {'SET_DISPLAY_ANALYTICS_DISPLAY_BANNER'} type
```
can we change this to be more relevant to what we're doing with the banner now? maybe `SET_DISPLAY_ANALYTICS_DISPLAY_BANNER` ?